### PR TITLE
James 2212 Fix Debian packaging

### PR DESCRIPTION
**Pull request by rouazana**
Replaces https://github.com/apache/james-project/pull/82

### CODE REVIEW

**Comment by aduprat** [comment]
Build triggered. sha1 is merged.

**Comment by aduprat** [comment]
Build started sha1 is merged.

**Comment by aduprat** [comment]
Build finished. No test results found.

**Comment by aduprat** [comment]
Build triggered. sha1 is merged.

**Comment by aduprat** [comment]
Build started sha1 is merged.

**Comment by aduprat** [comment]
Build finished. No test results found.

**Comment by aduprat** [comment]
Build triggered. sha1 is merged.

**Comment by aduprat** [comment]
Build started sha1 is merged.

**Comment by aduprat** [comment]
Build finished. No test results found.

**Comment by aduprat** [comment]
Build triggered. sha1 is merged.

**Comment by aduprat** [comment]
Build started sha1 is merged.

**Comment by aduprat** [comment]
Build finished. No test results found.

File: dockerfiles/compilation/java-8/Dockerfile [hunk 1]
```diff
@@ -8,7 +8,7 @@ ENV GIT_VERSION 1:2.1.4-2.1
 
 # Install Maven
 WORKDIR /root
-RUN wget http://mirrors.ircam.fr/pub/apache/maven/maven-3/3.5.0/binaries/apache-maven-3.5.0-bin.tar.gz
+RUN wget https://archive.apache.org/dist/maven/maven-3/3.5.0/binaries/apache-maven-3.5.0-bin.tar.gz
```
Comment (mbaechler) [inline dockerfiles/compilation/java-8/Dockerfile]: there's a trick for that, see what yann did for tika here : https://ci.linagora.com/linagora/lgs/openpaas/linagora.esn.tika/blob/master/docker-tikaserver/Dockerfile

Comment (rouazana) [reply dockerfiles/compilation/java-8/Dockerfile]: eurk. Do you think it's really needed? Ah you approved so I guess no :)

**Pull request merged**